package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "axe-linter-action",
+  "version": "1.0.0",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "ncc build src/index.ts --license licenses.txt",
+    "lint": "eslint .",
+    "fmt": "prettier --write \"src/**/*.{js,ts,tsx,json,md,css,html}\"",
+    "prepare": "husky",
+    "test": "mocha 'src/**/*.test.ts'",
```

Comment:
Just curious - why Mocha rather than `node:test`? I've been using the stdlib framework for the past few months (examples: [Launchpad](https://github.com/dequelabs/launchpad/blob/develop/packages/ui-issues/src/configuration.test.tsx#L1), [Trombone](https://github.com/dequelabs/trombone/blob/develop/api-client-js/src/Client.test.ts#L2)) and am very happy with it.